providers/src/airflow/providers/amazon/aws/auth_manager/aws_auth_manager.py

```diff
@@ -76,12 +76,12 @@ class AwsAuthManager(BaseAuthManager):
 
     Leverages AWS services such as Amazon Identity Center and Amazon Verified Permissions to perform
     authentication and authorization in Airflow.
-
-    :param appbuilder: the flask app builder
     """
 
-    def __init__(self, appbuilder: AirflowAppBuilder) -> None:
-        super().__init__(appbuilder)
+    appbuilder: AirflowAppBuilder | None = None
+
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
This change isn't backwards compatible with AF2.

Reply:
Yes, I'll make the AWS auth manager compatible only with Airflow 3. I dont think many users use it anyway.